modules/logging-loki-memberlist-ip.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-loki.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="logging-loki-memberlist-ip_{context}"]
+= Configuring Loki to tolerate memberlist creation failure
+
+In an OpenShift cluster, the administrators generally use a non-private IP network range. As a result, the LokiStack memberlist configuration fails as it only uses private IP networks.
+
+As an administrator, you can select the pod network for the memberlist configuration. You can modify the LokiStack CR to use the `podIP` in the `hashRing` spec:
+
+.Example LokiStack to include `podIP`
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki
+  namespace: openshift-logging
+spec:
+# ...
+  hashRing: <--- this and the next lines
```

Comment:
Probably, the comment in words `<--- this and the next lines` are not the best for being in the official documentation of Logging. 

A better option could replace the yaml example for using the command:
```
$ oc patch LokiStack logging-loki -n openshift-logging  --type=merge -p '{"spec": {"hashRing":{"memberlist":{"instanceAddrType":"podIP","type": "memberlist"}}}}'
```